### PR TITLE
Multiple regex patterns

### PR DESCRIPTION
**Pull request by arvind008**
Currently their is comparison of file names with file pattern using  String's matches() function.
It would be good to use Pattern.matches for ease of use.

### CODE REVIEW

File: platform/src/main/java/org/hillview/utils/Utilities.java [hunk 1]
```diff
@@ -54,11 +54,6 @@ public static String wildcardToRegex(@Nullable String wildcard) {
                 case '*':
                     s.append(".*");
                     break;
-                case '?':
```
Comment (mbudiu-vmw) [inline platform/src/main/java/org/hillview/utils/Utilities.java]: Why are you deleting this code?
The user types a *shell* pattern, and the Java code runs a regular expression. This code converts a shell pattern into a regular expression.

Comment (arvind008) [reply platform/src/main/java/org/hillview/utils/Utilities.java]: @mbudiu-vmw The shell pattern is interpreted as regular expression so conversion was not required. 

dot
question_mark

Comment (mbudiu-vmw) [reply platform/src/main/java/org/hillview/utils/Utilities.java]: it is a different regular expression.
In the shell ? means the same as dot: one arbitrary character.
As a regex it is a modifier meaning: the previous regex is optional.

**Comment by mbudiu-vmw** [comment]
I will close this PR for now. Note that we are still using file patterns and not regular expressions.
If you really want regular expressions you could add a checkbox to the input form allowing people to specify both regular expressions and filename patterns. Note that currently we only interpret patterns within the filename, but not within the directory name.

**Pull request closed without merging**